pkgs/by-name/we/weasis/package.nix

```
@@ -0,0 +1,85 @@
+{ lib
+, stdenv
+, fetchzip
+, jre
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+let
+  # Build process uses this naming for output
+  platform = if stdenv.isLinux then "linux"
+             else if stdenv.isDarwin then "macosx"
+             else if stdenv.isWindows then "windows"
+             else throw "unsupported platform";
```

This is generally written more clearly using an attrset:

```suggestion
  platform = {
    x86_64-linux
  }.${stdenv.system}
```